cohort-cli/src/main/java/com/ibm/cohort/cli/MeasureCLI.java

```diff
@@ -89,6 +89,9 @@
 		@Parameter(names = { "--disable-retrieve-cache" }, description = "Disable the use of the retrieve cache.")
 		private boolean disableRetrieveCache = false;
 
+//		@ParametersDelegate
```

Comment:
Is this commented code prep work for the next task?

Reply:
Yes. It was more so to provide an example of usage (0e84a19).
I can revert the commit if there are objects to having it in the codebase,
but the uncommented version will likely find its way into code at some point 😉